### ISSUE TITLE
Test navigation get test site flow

### ISSUE THREAD

**Issue by fparga** [dialoguemd/covidflow#282]
Implement first version of the get test site flow. 

* Response will be displayed in a carousel.
* For now assume the user will provide its postal code, support for GPS coordinates sent as metadata will be added later.
* For now assume no image in the response. Map image will be added later once #276 is done.
* The Call button should be a simple `tel:` url.
* The Destination button should be a link to maps with either GPS coordinates or the address of the testing site (depending on the information we get from Clinia API). The use of apple maps URL seems to do a better job than Google Maps on most devices, redirecting the user to Google Maps when appropriate, we have to test it on additional devices/browser though. [Map Links Scheme](https://developer.apple.com/library/archive/featuredarticles/iPhoneURLScheme_Reference/MapLinks/MapLinks.html) 
* The more information button should send an intent to which the bot will reply with the address and opening hours of the site. It's probably best to implement this part in another PR and to leave out this button for now.

Here is a sample of the message format expected by the widget:

```
dispatcher.utter_message(
    attachment={
        "type": "template",
        "payload": {
            "template_type": "generic",
            "elements": [
                {
                    "title": "Test site name",
                    "subtitle": "Test site description",
                    "image_url": "http://....",
                    "buttons": [
                        {
                            "title": "Call (514) 555-1234",
                            "type": "web_url",
                            "url": "tel:514-555-1234",
                        },
                        {
                            "title": "Destination",
                            "type": "web_url",
                            "url": "http://maps.apple.com/?q=51.507269,-0.127695",
                        },
                        {
                            "title": "More information",
                            "type": "postback",
                            "payload": "/an.intent",
                        },
                    ],
                },
                ...
            ],
        },
    }
)
```

**Comment by fparga** [comment]
Blocked by #265, #268
